packages/client/src/components/app/forms/CodeScanner.svelte

```diff
@@ -114,6 +115,12 @@
     oscillator.start()
     oscillator.stop(endTime)
   }
+
+  const handleChange = e => {
```

Comment:
I think you can remove all of these other changes to CodeScanner.svelte - it looks like the event dispatcher is firing the onChange